### ISSUE TITLE
ST3: Disabling keybindings doesn't work

### ISSUE THREAD

**Issue by pmorch** [mikefowler/simple-clone#8]
I wanted to disable the "right" keybinding, but it doesn't work in ST3, even though the exact same procedure works for me in ST2.

I've created a `$HOME/.config/sublime-text-3/Packages/User/SimpleClone.sublime-settings` file (on Linux) containing

```
{ "disabled_keymaps": "all" }
```

But `ctrl+shift+right` still clones. Creating the exact same file for ST2 (`$HOME/.config/sublime-text-3/Packages/User/SimpleClone.sublime-settings`) works like a charm.

The only workaround I could use: Install the package in a directory and edit `$HOME/.config/sublime-text-3/Packages/SimpleClone/Default (Linux).sublime-keymap` to remove the binding manually.

**Comment by mikefowler** [comment]
Interesting. The solution to this isn't immediately apparent to me, but I'll investigate when I have some time. In the meantime if you discover a more elegant solution, feel free to send a pull request my way.

**Comment by ddauer** [comment]
Same issue here on OS X. Overriding the key bindings doesn't work...

**Comment by artivilla** [comment]
Yup, doesn't work. I'm using OS X Mavericks. Trying to disable the right key.

**Comment by mikefowler** [comment]
Thanks for reporting all. I'm fairly busy with other projects at the moment, so if anybody wants to take a shot at a bugfix I'll happily merge your pull request. Thanks!

**Comment by fhemberger** [comment]
PR #9 doesn't seem to fix the issue, had to extract the package and change `Default (OSX).sublime-keymap` manually.

**Comment by karlobermeyer** [comment]
When the { "disabled_keymaps": "all" } thing didn't work for me, with Sublime Text 3 on Max OS X 10.10.5, I was able to restore selection key bindings by going to Preferences -> Keybindings - User, and entering
[
    { "keys": ["shift+command+left"], "command": "move_to", "args": {"to": "bol", "extend": true } },
    { "keys": ["shift+command+right"], "command": "move_to", "args": {"to": "eol", "extend": true } },
    { "keys": ["shift+command+up"], "command": "move_to", "args": {"to": "bof", "extend": true } },
    { "keys": ["shift+command+down"], "command": "move_to", "args": {"to": "eof", "extend": true } }
]
